config.json

```diff
@@ -69,7 +69,8 @@
     },
     "travis": {
       "blacklist": [
-        "repos/polymer-starter-kit"
+        "repos/polymer-starter-kit",
+        "repos/paper-checkbox"
```

Comment:
@azakus oops! Right! I had done that because I had a problem with the editor. Updated! Thanks!